examples/angular/unit-testing/jest.config.ts

```
@@ -0,0 +1,5 @@
+const config: import('jest').Config = {
```

ESLint

[error] `import()` type annotations are forbidden.

(@typescript-eslint/consistent-type-imports)